### ISSUE TITLE
Feature Request: ARM compatibility

### ISSUE THREAD

**Issue by svh1985** [cristianbuse/VBA-FileTools#23]
Hi there,

I'm running the ARM version of Windows & Office and my Excel projects using the VBA-FileTools are showing a lot of errors.
While I'm doing my best to fix them, I'm not a VBA expert.

Hoping to get some tips and support!

One thing I noticed is that a lot of VBA functions need to be changed.
Example: LeftB$ changed to VBA.LeftB$.

Thanks!

**Comment by cristianbuse** [comment]
Hi @svh1985 ,

Thanks for letting me know!

None of my machines are on ARM so I cannot test. I will try to install a virtual machine when I get some time in the next few weeks and will get back to you.

**Comment by cristianbuse** [comment]
@svh1985 ,

Is there any chance that you have an addin or different code module that has a ```LeftB``` function defined?
In other words, what kind of error do you get? For example:
![image](https://github.com/cristianbuse/VBA-FileTools/assets/23198997/16e2cb62-5e56-4472-b032-4589bbf1d765)
or
![image](https://github.com/cristianbuse/VBA-FileTools/assets/23198997/5d523c8a-7a7d-4b88-a06b-690a560a7bf5)

**Comment by svh1985** [comment]
Hi @cristianbuse,

I read on issue https://github.com/cristianbuse/VBA-FileTools/issues/21 that the user was running on Win11 ARM too, and only had a minor unrelated issue. So, I figured something else might be wrong in my case. Therefore, I compared the references of an empty Excel file with my project and did some corrections. And it seems to be work now!

Thanks for helping out, and your efforts on this project!

_Ps. I think ARM can be added to the list supported CPU’s._
